arch/arm64/src/imx9/imx9_flexspi_nor.c

```diff
@@ -790,12 +790,15 @@ static ssize_t imx9_flexspi_nor_read(struct mtd_dev_s *dev,
     }
 
   src = priv->ahb_base + offset;
-  DEBUGASSERT(((uintptr_t)src & ALIGN_MASK) == 0);
 
   up_invalidate_dcache((uintptr_t)buffer,
                        (uintptr_t)buffer + ALIGN_UP(nbytes));
+  int n = nbytes;
 
```

Comment:
This is fine, but what is the cache invalidate above about???

Reply:
done, removed